tests/visualize_test.py

```diff
@@ -222,6 +265,48 @@ def test_heatmap_archive__grid(grid_archive):
     plt.figure(figsize=(8, 6))
     grid_archive_heatmap(grid_archive)
 
+@image_comparison(baseline_images=["grid_archive_heatmap_equal_aspect"],
+                  remove_text=False,
+                  extensions=["png"])
+def test_heatmap_archive__grid_equal_aspect(grid_archive):
+    plt.figure(figsize=(8, 6))
+    grid_archive_heatmap(grid_archive, aspect="equal")
+
+@image_comparison(baseline_images=["grid_archive_heatmap_aspect_gt_1"],
+                  remove_text=False,
+                  extensions=["png"])
+def test_heatmap_archive__grid_aspect_gt_1(grid_archive):
+    plt.figure(figsize=(8, 6))
+    grid_archive_heatmap(grid_archive, aspect=2.5) # random aspect
```

Comment:
No need to mention that it's a random aspect as the test name already shows this (same comment below too)